colrev/ops/built_in/prescreen/scope_prescreen.py

```diff
@@ -185,8 +185,20 @@ def __conditional_prescreen_complementary_materials(
                 ):
                     record.prescreen_exclude(reason="complementary material")
 
+    def __conditional_presecreen_not_in_ranking(
+        self, record: colrev.record.Record
+    ) -> None:
```

Comment:
This part of the prescreen should only run when a `self.settings.Ranking` is set (similar to the other `__conditional_prescreen*` methods).